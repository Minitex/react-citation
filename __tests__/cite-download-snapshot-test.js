import React from 'react'
import renderer from 'react-test-renderer'
import CitationDownload from '../src/cite-download'

describe('CitationDownload Tests', () => {
  it("produces download links", function(){  

    const fields = 
      [
        {
          src: 'http://lib-mdl-dev.oit.umn.edu/thumbnails/irrc:1744',
          url: 'http://reflections.mndigital.org/utils/getstream/collection/irrc/id/1744'
        },
        {
          src: 'http://lib-mdl-dev.oit.umn.edu/thumbnails/irrc:1746',
          url: 'http://reflections.mndigital.org/utils/getstream/collection/irrc/id/1746'
        },
        {
          src: 'http://lib-mdl-dev.oit.umn.edu/thumbnails/irrc:1747',
          url: 'http://reflections.mndigital.org/utils/getstream/collection/irrc/id/1747'
        }
      ]

    let component = renderer.create(<CitationDownload fields={fields} />)
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})