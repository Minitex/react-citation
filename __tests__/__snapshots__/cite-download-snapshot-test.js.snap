exports[`CitationDownload Tests produces download links 1`] = `
<div>
  <div
    className="row">
    <div
      className="col-md-2">
      <a
        className="thumbnail"
        hgref="http://reflections.mndigital.org/utils/getstream/collection/irrc/id/1744">
        <img
          src="http://lib-mdl-dev.oit.umn.edu/thumbnails/irrc:1744" />
        <span
          className="glyphicon glyphicon-download-alt" />
         
        Download
      </a>
    </div>
  </div>
  <div
    className="row">
    <div
      className="col-md-2">
      <a
        className="thumbnail"
        hgref="http://reflections.mndigital.org/utils/getstream/collection/irrc/id/1746">
        <img
          src="http://lib-mdl-dev.oit.umn.edu/thumbnails/irrc:1746" />
        <span
          className="glyphicon glyphicon-download-alt" />
         
        Download
      </a>
    </div>
  </div>
  <div
    className="row">
    <div
      className="col-md-2">
      <a
        className="thumbnail"
        hgref="http://reflections.mndigital.org/utils/getstream/collection/irrc/id/1747">
        <img
          src="http://lib-mdl-dev.oit.umn.edu/thumbnails/irrc:1747" />
        <span
          className="glyphicon glyphicon-download-alt" />
         
        Download
      </a>
    </div>
  </div>
</div>
`;
